src/server/routing/router-development.ts

```diff
@@ -83,20 +86,25 @@ authRouterDevelopment.get(
       sameSite: 'lax',
     };
     const authMethod = req.params.authMethod as AuthProfile['authMethod'];
+    const testAccounts =
+      authMethod === 'digid' ? testAccountsDigid : testAccountsEherkenning;
+    const allUsernames = Object.keys(testAccounts);
     const userName =
       req.params.user && req.params.user in testAccounts
         ? req.params.user
-        : Object.keys(testAccounts)[0];
+        : allUsernames[0];
```

Comment:
waarom de eerste?

Reply:
Gemak. In dev is er standaard maar 1 user. Dan hoef je niet een extra klik te maken voor inloggen.